### PR TITLE
Add branch power calculation explanation in docs

### PR DESCRIPTION
**Pull request by nitbharambe**

### CODE REVIEW

File: docs/user_manual/calculations.md [hunk 1]
```diff
@@ -125,6 +125,26 @@ Output:
...
+    \begin{eqnarray}
+        \underline{S_{branch-side}} = \sqrt{3} \cdot \underline{U_{node-line-line-side}} \cdot \underline{I_{branch-side}}
+    \end{eqnarray}
+$$
+
+These quantities are in complex form. Hence, they can be constructed by PGM output attributes in the following way:
+
+For  $\underline{U}$ of nodes, `u`, `u_angle` can be used.
+
+For  $\underline{I}$ of branches, `i_side`, `p_side`, `q_side` can be used: $\underline{I} = i_{side} \angle \arctan(((p_{side} + j \cdot q_{side}) / \underline{U})*)$.
```
Comment (mgovers) [inline docs/user_manual/calculations.md:142]: a lot of nested braces. can you use `\frac` to pretty it up a little bit? also:
* please double-check that the `*` works as intended, it's not superscripted `^*`
* the `\angle` feels out-of-place here. `\arctan` already implies `\angle`. or you can just make it `\exp i ...`. even if I understand what you mean, at the very least, it's a notation i've never seen before.

File: docs/user_manual/calculations.md [hunk 2]
```diff
@@ -125,6 +125,26 @@ Output:
...
+
+The `side` here can be `from`, `to` for `Branch`-es, `1`, `2`, `3` for `Branch3`-s.
```
Comment (mgovers) [inline docs/user_manual/calculations.md:144]:
```suggestion
The `side` here can be `from`, `to` for `Branch`es, `1`, `2`, `3` for `Branch3`s. 
```

File: docs/user_manual/calculations.md [hunk 3]
```diff
@@ -125,6 +125,23 @@ Output:
...
+    \begin{eqnarray}
+        \underline{S_{branch-side}} = \sqrt{3} \cdot \underline{U_{LL-side-node}} \cdot \underline{I_{branch-side}}
+    \end{eqnarray}
+$$
+
+These quantities are in complex form. Hence, they can be constructed by PGM output attributes in the following way:
+
+* For  $\underline{U}$ of nodes, `u` is the magnitude and `u_angle` is the angle. Also the line to neutral voltage can be converted into line to line voltage by $ U_{LN} = U_{LL} / \sqrt{3}$. Check [Node Steady State Output](components.md#steady-state-output) to find out which quantity is relevant in your calculation.
+
+* For  $\underline{I}$ of branches, `i_side` is the maginute and angle can be found from `p_side` and `q_side` by: $\arctan(\frac{P_{side} + j \cdot Q_{side}}{\underline{U}})^{*}$.
```
Comment (mgovers) [inline docs/user_manual/calculations.md:142]:
```suggestion
* For  $\underline{I}$ of branches, `i_side` is the magnitude. Its angle can be found from `p_side` and `q_side` by: $\arctan(\frac{P_{side} + j \cdot Q_{side}}{\underline{U}})^{*}$.
```

File: docs/user_manual/calculations.md [hunk 4]
```diff
@@ -125,6 +125,23 @@ Output:
...
+The `side` here can be `from`, `to` for `Branch`es, `1`, `2`, `3` for `Branch3`s.
```
Comment (mgovers) [inline docs/user_manual/calculations.md:143]: please add a hyperlink to the `Branch` and `Branch3`

Comment (nitbharambe) [reply docs/user_manual/calculations.md:143]:
```suggestion
The `side` here can be `from`, `to` for {hoverxreftooltip}`user_manual/components:Branch`es, `1`, `2`, `3` for `{hoverxreftooltip}`user_manual/components:Branch3`s. 
```

**Review by mgovers — APPROVED** [review]

**Pull request merged**